hasher-matcher-actioner/src/OpenMediaMatch/storage/postgres/impl.py

```diff
@@ -566,11 +566,14 @@ def bank_add_content(
         sesh.commit()
         return content.id
 
-    def bank_remove_content(self, bank_name: str, content_id: int) -> None:
-        database.db.session.execute(
-            delete(database.BankContent).where(database.BankContent.id == content_id)
+    def bank_remove_content(self, bank_name: str, content_id: int) -> int:
+        result = database.db.session.execute(
+            delete(database.BankContent)
+            .where(database.BankContent.id == content_id)
+            .where(database.BankContent.imported_from_id.is_(None))
         )
```

Comment:
blocking: Hmm, not sure about this change - would rather it throw a exception rather than silently fail.

I think you can leave this off for now and just add a TODO to throw the exception - the bank as a whole should be marked as import or not so we shouldn't need to check on a row-by-row basis.

Reply:
got it, will update!